core/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -355,7 +355,8 @@ class Router(formatOps: FormatOps) {
           case _ => NoSplit
```

Comment:
Here we want to add a case that produces a `Space` that fixes issue #618. A good way to debug what's happening here is to print out `leftOwner` (which is a [scala.meta.Tree](http://scalameta.org/tutorial/#Part2-Trees))

```scala
case _ =>
  logger.elem(leftOwner.syntax, leftOwner.structure)
  NoSplit
```